scripts/test/watch

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh
-exec filewatcher -L 6 -x build -x script go test -timeout 10s -v './${dir}'
+# shellcheck disable=SC2154
+exec filewatcher -L 6 -x build -x script go test -timeout 10s -v "./${dir}"
```

Comment:
I wasn't sure for that since dir wasn't declared. 

The shellcheck whitelist code is actually wrong here. It will need to be present as:

`#shellcheck disable=SC2016` when I roll back to single quotes.